GeofenceDisplay/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "module": "ESNext",
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "skipLibCheck": true,
+    "allowJs": true,
```

Comment:
convert rastercoords to ts to remove the need for this

Reply:
Will do